library/NotificationCenter/Queue/QueueManager.php

```
@@ -162,12 +161,15 @@ protected function storeFiles(GatewayInterface $gateway, QueuedMessage $queuedMe
 
         // Copy the attachments to the temporary folder
         foreach ($attachments as $index => $originalPath) {
-            $originalPath = str_replace(TL_ROOT . '/', '', $originalPath);
-            $clonePath = $folder->path . '/' . basename($originalPath);
+            $originalPath = '/' === $originalPath[0] ? $originalPath : str_replace(TL_ROOT . '/', '', $originalPath);
+            $clonePath = TL_ROOT . '/' . $folder->path . '/' . basename($originalPath);
 
             // Update the tokens if copy was successful
-            if (Files::getInstance()->copy($originalPath, $clonePath)) {
-                $attachments[$index] = TL_ROOT . '/' . $clonePath;
+            try {
+                (new Filesystem())->copy($originalPath, $clonePath);
+                $attachments[$index] = $clonePath;
+            } catch (\Exception $exception) {
+                // noop
```

I don't think it should silently fail?